File: LMG.Fab.Web/ClientApp/app/models/lot.ts
export class Lot {
    PkLot: number;
    NomLot: string;
    FkParamDetailProc: number;
    CodeLot: string;
    FkOffices?: number;
    DateArriveeLivres?: Date;
    DateMiseEnVente?: Date;
    NomProcessus: string;
    CodeProcessus: string;
    IsSelected: boolean = false;
    EnCours: boolean
}

File: LMG.Fab.Data/Entities/LmgFab/Lot.cs
using System;
using System.Collections.Generic;

namespace LMG.Fab.Data.Entities.LmgFab
{
    public partial class Lot
    {
        public int PkLot { get; set; }
        public string NomLot { get; set; }
        public int? FkParamDetailProc { get; set; }
        public string CodeLot { get; set; }
        public int? FkOffices { get; set; }
        public DateTime? DateArriveeLivres { get; set; }
        public DateTime? DateMiseEnVente { get; set; }
        public bool? EnCours { get; set; }

        public ParamDetail FkParamDetailProcNavigation { get; set; }
    }
}
